draft-ietf-acme-acme.md

```diff
@@ -2804,6 +2880,10 @@ interoperable definition and does not overlap with existing validation methods.
 That is, it should not be possible for a client and server to follow the
 same set of actions to fulfill two different validation methods.
 
+The values "tls-sni-01" and "tls-sni-02" are reserved because they
+were used in pre-RFC versions of this specification to denote
+validation methods that were found not to be secure in some cases.
```

Comment:
I think it would be worth adding "and were removed from the specification" to the end of this paragraph.